### PR TITLE
Update Swedish translation

### PR DESCRIPTION
**Pull request by NickWick13**
Update sv.po

### CODE REVIEW

File: packages/tools/locales/sv.po [hunk 1]
```diff
@@ -5577,16 +5570,3 @@ msgstr "Zooma in"
...
-
-#, fuzzy
-#~ msgid "%s: Missing password"
-#~ msgstr "Huvudlösenord"
-
-#~ msgid "Share and collaborate on a notebook"
-#~ msgstr "Dela och samarbeta i en anteckningsbok"
-
-#~ msgid "Sharing access control"
-#~ msgstr "Dela åtkomstkontroll"
```
Comment (tessus) [inline packages/tools/locales/sv.po]: please don't delete these.

**Review by tessus — CHANGES_REQUESTED** [review]
see my inline comment

**Comment by NickWick13** [comment]
@tessus I did as you said

**Review by tessus — APPROVED** [review]
Thanks for your update.

**Pull request merged**